pkg/printers/internalversion/printers.go

```diff
@@ -472,7 +472,7 @@ func AddHandlers(h printers.PrintHandler) {
 
 	endpointSliceColumnDefinitions := []metav1beta1.TableColumnDefinition{
 		{Name: "Name", Type: "string", Format: "name", Description: metav1.ObjectMeta{}.SwaggerDoc()["name"]},
-		{Name: "AddressType", Type: "string", Description: discoveryv1alpha1.EndpointSlice{}.SwaggerDoc()["addressType"]},
+		{Name: "Address Type", Type: "string", Description: discoveryv1alpha1.EndpointSlice{}.SwaggerDoc()["addressType"]},
```

Comment:
I think these were targeted as no space items to make awk easier

Reply:
That makes sense, thanks! I wasn't completely sure as it looked like there was some of both - switched back to no space.